charts/alfresco-transform-service/templates/_helpers.tpl

```diff
@@ -40,6 +40,7 @@ helm.sh/chart: {{ include "alfresco-transform-service.chart" . }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+app.kubernetes.io/component: {{ template "alfresco-transform-service.name" . }}
```

Comment:
add some tests for this chart because I imagine it has the same problem (using the overridden name by the custom labals named template) I've just spotted before